### DISCUSSION TITLE
docs/localization/joining-the-localization-team/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#195] [Page comments]
# docs/localization/joining-the-localization-team/

Thanks for being interested on volunteering to translate Scratch Addons to other languages! Scratch Addons is a non-profit open source project where volunteer web developers maintain the extension and create its addons.

https://scratchaddons.com/docs/localization/joining-the-localization-team/

**Comment by ghost** [comment]
Can i help translating for The netherlands?
I come from The netherlands anyways


~~?????????~~

**Reply by Norbiros** [reply]
Yes you can! Just join us on transifex :)

**Reply by ghost** [reply]
what is _transifex_?

`<html>
<p> i dont get it </p>
</html>`

**Reply by Norbiros** [reply]
https://www.transifex.com/scratch-addons/scratch-addons-extension/

**Reply by ghost** [reply]
i asked for netherlands-Belgium so i can Join.

**Comment by zodicttv** [comment]
do you know how to translate into english?
im on my cousins computer and its in the nederlands

**Reply by CST1229** [reply]
SA is already translated into English (it's its primary language!). To change SA's language, you have to change your browser's language.

**Reply by ghost** [reply]
I know! i can help you if you want :)

**Comment by PJPETKO** [comment]
hello

**Comment by FnafandPvzPro** [comment]
If it is ok I can translate Slovakian.

**Comment by cesou91** [comment]
I asked to translate it in French because I've found some grammar errors...

Can you accept me ?
I am cesou91_officiel

**Reply by WorldLanguages** [reply]
Accepted!

**Reply by cesou91** [reply]
The word "on" means "dans" and "sur" in French but in the glossary we cannot put other translations :/

**Comment by 1virez** [comment]
I could translate it into german since its my main language

**Reply by ParrotGames-Dev** [reply]
cool B-)

**Comment by ParrotGames-Dev** [comment]
I would like to help with translation on Russian.

**Reply by ParrotGames-Dev** [reply]
Is that auto-upwote?

**Reply by ParrotGames-Dev** [reply]
Oops, I mean upVote

**Reply by Joanne31012** [reply]
No~~It's not auto-upvote.

**Comment by Joanne31012** [comment]
可以運用Google翻譯,再加以改正不流順的句子或單字，翻譯是一件簡單的事，只要有心想做就做得到!
(You can use Google Translate, and then correct the sentences or words that are not fluent, translation is a simple thing, as long as you want to do it, you can do it!)

**Reply by cobaltt7** [reply]
Please don't use Google translate, it's often incorrect.

**Reply by Hans5958** [reply]
@RedGuy12 Eh, she said to "correct the sentences or words that are not fluent" after that. It is fine to use translator **as a reference**.

**Comment by mimikunda** [comment]
How fast do you accept people (im Mimikunda if you can get me in)

**Comment by realunicat** [comment]
Can you make it so you can translate to all supported languages? I don't wanna just right-click and choose the "Translate to English" option.

**Comment by PJPETKO** [comment]
Hi

**Reply by WorldLanguages** [reply]
Hello!

**Comment by kostyaiscool** [comment]
Can i please translate new addons to russian, and translate addons to ukrainian?

**Reply by kostyaiscool** [reply]
My name in Scratch is s_ecialkai

**Reply by WorldLanguages** [reply]
Please follow the steps and request to join inside Transifex

**Comment by Austin819-Master** [comment]
Addons request: Follow button to own profile and Scratch 3.0 to 2.0. (The Scratch 3.0 to 2.0 will make Scratch 3.0 styled pages look like 2.0 (The 2.0 styled pages that look like 3.0 will not count))

**Reply by Austin819-Master** [reply]
My name in Scratch is Austin819

**Reply by Hans5958** [reply]
This is not the place. You can suggest on https://github.com/ScratchAddons/ScratchAddons/discussions instead.

Anyways, this is already been suggested on ScratchAddons/ScratchAddons#985. You should check it out instead.

**Comment by broreally573** [comment]
How do I find The Record Button In Scratch addons V1.33.2? I have Project Video Recorder On but when the update released I now can't find it. Can someone help?

**Reply by WorldLanguages** [reply]
For now, try disabling the "block count" and "hide tutorials button" addons, if you have them enabled. That should fix the issue.

**Comment by Durian434** [comment]
can you translate website too?

**Reply by Durian434** [reply]
(mean translate the part of website that addon add it)

**Reply by WorldLanguages** [reply]
Yes, most parts of the scratchaddons.com website can be translated.

**Reply by CST1229** [reply]
And also everything that addons add to the Scratch website can be translated (I assume that's what they meant).

**Comment by HoangKhangCoder** [comment]
Can I help translating for Vietnamese?
I come from Vietnam

**Comment by Jessica-Jannat** [comment]
Hello! I also joined Transifex, and my scratch account is "BD_CountryBall" means "BangladeshCountryBall" ,  I want to translate it into Bengali, can I?

**Comment by Emil0717** [comment]
Сan i make more Russian translate? Some texts aren't translated (username: Emil0717)
